### PR TITLE
Task s5 popup

### PR DESCRIPTION
**Pull request by C0deJack**
creation of a popup to display winner

### CODE REVIEW

File: css/popup.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    height: 300px;
+    position: absolute;
+    padding: 30px;
+    left: 100px;
+    top: 100px;
+    border-radius: 5px;
+    background-color: #b3ff0f;
+}
+
+.restart h2  {
```
Comment (o-tahboub) [inline css/popup.css]: Two spaces after h2, one is used in above selectors.

File: css/popup.css [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    text-align: center;
+    font-size: 2rem;
+}
+
+.restart h1  {
```
Comment (o-tahboub) [inline css/popup.css]: Two spaces after h2, one is used in above selectors.

File: css/popup.css [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    text-align: center;
+    font-size: 5rem;
+}
+
+#btnRestart {
```
Comment (o-tahboub) [inline css/popup.css:23]: Can a class be used here, id's are not recommended for css selections.

File: css/popup.css [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    font-size: 1.6rem;
+    position: absolute;
+    width: 300px;
+    left: 240px;
+    top: 250px;
+    border: none;
+    background-color: #b3ff0f;
+}
+
+#btnRestart:hover {
```
Comment (o-tahboub) [inline css/popup.css:34]: Can a class be used here, id's are not recommended for css selections.

File: css/popup.css [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+    cursor: pointer;
+    color: #ededed;
+    text-shadow: 2px 2px 2px #000000, 2px -2px 2px #000000, -2px -2px 2px #000000, -2px 2px 2px #000000;
+}
+
+#btnRestart:active {
```
Comment (o-tahboub) [inline css/popup.css:40]: Can a class be used here, id's are not recommended for css selections.

File: css/popup.css [hunk 6]
```diff
@@ -0,0 +1,60 @@
+* {
+    box-sizing: border-box;
```
Comment (o-tahboub) [inline css/popup.css]: This already exists in styles.css

File: css/popup.css [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+}
+
+body {
+    font-family: 'Press Start 2P', cursive;
```
Comment (o-tahboub) [inline css/popup.css]: This might be better added to styles.css.

File: js/popup.js [hunk 8]
```diff
@@ -0,0 +1,24 @@
+$('.gameLevel').on('collision', function (player) {
```
Comment (o-tahboub) [inline js/popup.js:1]: Function incomplete, this might be due to dependencies.

**Review by o-tahboub — COMMENTED** [review]
Please see comments.

File: css/popup.css (hunk 1 above)
Comment (C0deJack) [reply css/popup.css]: changed

File: css/popup.css (hunk 2 above)
Comment (C0deJack) [reply css/popup.css]: done

File: css/popup.css (hunk 3 above)
Comment (C0deJack) [reply css/popup.css:23]: The button keeps inheriting the css from the page that its loaded into so i've tagged it to the id as the id is being used by the javascript query selector

File: css/popup.css (hunk 4 above)
Comment (C0deJack) [reply css/popup.css:34]: as above

File: css/popup.css (hunk 5 above)
Comment (C0deJack) [reply css/popup.css:40]: as above

File: css/popup.css (hunk 6 above)
Comment (C0deJack) [reply css/popup.css]: removed duplication

File: css/popup.css (hunk 7 above)
Comment (C0deJack) [reply css/popup.css]: removed duplication

File: js/popup.js (hunk 8 above)
Comment (C0deJack) [reply js/popup.js:1]: function is ready for integration into collision detection code not yet written.

**Review by o-tahboub — APPROVED** [review]
Spaces and generic CSS was amended, Id's were confirmed necessary.

**Pull request merged**